scripts/init.js

```diff
@@ -261,6 +263,7 @@ function main() {
               { title: "Android", value: "android", selected: true },
               { title: "iOS", value: "ios", selected: true },
               { title: "macOS", value: "macos", selected: true },
+              { title: "visionOS (Experimental)", value: "visionos", selected: true },
```

Comment:
I added `(Experimental)` here as a disclaimer.

Reply:
Should probably be unselected by default:
```suggestion
              { title: "visionOS (experimental)", value: "visionos", selected: false },
```